### DISCUSSION TITLE
Top left hand corner

### DISCUSSION THREAD

**Discussion by 1a2a3a1q2s3c** [MMRIZE/MMM-CalendarExt3#115] [Q&A] [answered]
Hello, the top left hand corner is defaulted for the calendar module right?
Is it possible to change them to show like how cal ext2  is showing whereby it’s showing events for “today” and the next x number of days?

currently, in order to achieve this, I’m using both ext 3 and ext 2.

**Comment by eouia** [comment, marked as answer]
mode:”day”
The difference from CX2 is, this will show 7days, not random N days.

**Reply by 1a2a3a1q2s3c** [reply]
The current default is showing US calendar. So does that mean I change the US cal to Google cal and input mode to day and it will show today and the next 7 days?

**Reply by eouia** [reply]
Yes. You can expand the range with “weeksInView”. 1 for 7days, 2 for 14days and so on…

**Reply by eouia** [reply]
And if you want, with “dayIndex”, you can adjust start day. For example; -1 would mean starting from Yesterday instead of Today.

**Reply by 1a2a3a1q2s3c** [reply]
cant get it to work... it is showing header 'today' with a bunch of events that follows for the next few weeks. it goes like this -
**today**
go dental 20 nov
go doctor 30 nov
go market 2 dec

which kinda doesnt make sense.

i was hoping to get this->
**today**
go dental 18 nov
**next 6 days**
go somewhere 19 nov (if no entries in calendar then empty)
go somewhere 20 nov (if no entries in calendar then empty)
go somewhere 21 nov (if no entries in calendar then empty)
go somewhere 22 nov (if no entries in calendar then empty)
go somewhere 23 nov (if no entries in calendar then empty)
go somewhere 24 nov (if no entries in calendar then empty)


what am i doing wrong here?

{
                        module: "calendar",
                        broadcastPastEvents: true,
                        header: "Today",
                        position: "top_left",
                        mode: "day",
                        config: {
                                calendars: [
                                        {
                                         url: "123"
                                        name: "123",
                                        },
                                        ]
}
},

**Reply by eouia** [reply]
You are saying "calendar" module(The original default calendar module). Mine is "MMM-CalendarExt3". I misunderstood your question. sorry.

**Reply by 1a2a3a1q2s3c** [reply]
But your ext3 needs to work with the default calendar module right? I’m using your ext3 module along with your default calendar module. Is it possible to get it to work like how I wanted it?

**Reply by 1a2a3a1q2s3c** [reply]
this is my full code

{
                         module: "MMM-CalendarExt3",
                         position: "bottom_bar",
                         title: "",
                         config: {
                          mode: "weekly",
                          instanceId: "basicCalendar",
                          weekIndex: 0,
                          weeksInView: 4,
                          maxEventLines: 6,
                          fontSize: '18px',
                          eventHeight: '13px',
                          firstDayOfWeek: 0,
                          useMarquee: true,
                          calendarSet: [],
    }
                },

{
module: "calendar",
broadcastPastEvents: true,
header: "Today",
position: "top_left",
mode: "day",
config: {
calendars: [
{
url: "123"
name: "123",
},
]
}
},

**Reply by eouia** [reply]
I'm not sure what you mean `events for “today” and the next x number of days`.
If what you mean is limit the days of events to N, it will be `maximumNumberOfDays` of the default calender config.

But, what you want is something like this? (orange lined box)
![ttt](https://github.com/MMRIZE/MMM-CalendarExt3/assets/1720610/a1ff8df0-dc2d-4080-afb7-9d9d3f471b8f)
It is MMM-CalendarExt3Agenda, the sibling module of CX3.

**Reply by 1a2a3a1q2s3c** [reply]
yes exactly that the orange line box at the top left hand corner and the full layout calendar at the bottom. how do i do that?

**Reply by 1a2a3a1q2s3c** [reply]
how do i remove the mini calendar? i tried showMiniMonthCalendar: false, but its not working

**Reply by eouia** [reply]
It should work.
```js
{
	module: "MMM-CalendarExt3Agenda",
	position: "top_left",
	config: {
		showMiniMonthCalendar: false,
                 ...
	}
},
```

**Reply by 1a2a3a1q2s3c** [reply]
got it now! i put that statement before config! silly me!
i think im getting there now. managed to get ext3agenda at top left and ext3 full cal at bottom bar. is there a way to change color? i realise when google photo shows bright pictures, i cant see any of the fonts.

**Reply by eouia** [reply]
Give background balck or darker
```css
/* css/custom.css */
.MMM-CalendarExt3,
.MMM-CalendarExt3Agenda {
  padding: 10px;
  border-radius: 10px;
  background-color: rgb(0 0 0 / 75%);
}
```
<img width="536" alt="image" src="https://github.com/MMRIZE/MMM-CalendarExt3/assets/1720610/9c8ec580-6797-4fea-8ad6-78cdc9a29de9">

**Reply by 1a2a3a1q2s3c** [reply]
Thank you so much for your patience! i would have never thought of that.
